go.mod

```diff
@@ -101,3 +101,5 @@ require (
 )
 
 go 1.18
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/mfrancisc/toolchain-common v0.0.0-20230105093252-b18c1ea3a028
```

Comment:
reminder

Reply:
changed in [08f6500](https://github.com/codeready-toolchain/member-operator/pull/403/commits/08f650004ec1972fa8d610cf26d596c9044a9351)